symfony/src/EventListener/RequestListener.php

```diff
@@ -73,7 +73,8 @@ private function checkPageNotices()
 
     private function checkSiteSelect()
     {
-        if (!$this->session->has('site') && !preg_match('/^(\/s)?\/(_profiler|_wdt|cron|admin|help|settings|problem|biobank|review)($|\/).*/', $this->request->getPathInfo())) {
+        if (!$this->session->has('site') && !preg_match('/^(\/s)?\/(_profiler|_wdt|cron|admin|help|settings|problem|biobank|review|workqueue)($|\/).*/',
```

Comment:
I added the `review` one recently, but I can't remember -- should this be accessible if a site isn't set?

Reply:
Had to add this to enable super users (stsi) to access WorkQueue. Yes, this is not accessible if a site is not set. In the WorkQueue controller there is a check for site's awardee ([Line 66](https://github.com/all-of-us/healthpro/blob/sn/hpro-753/migrate-workqueue-to-symfony/symfony/src/Controller/WorkQueueController.php#L66)) and renders `no-organization.html.twig` file if it's empty (site is not set)